File: trunk/FlexTermShell/interfaces/ISSH2ClientService.php
<?php
require_once (dirname(dirname(__FILE__)).'/dao/SSH2DAO.php');
/**
 *
 */
interface ISSH2ClientService 
{
	function tryConnectThenExec(SSH2DAO $ssh2dao);
	
	function tryConnect(SSH2DAO $ssh2dao);
	function tryLogin(SSH2DAO $ssh2dao);
	function tryExec(SSH2DAO $ssh2dao);
}

?>

File: FlexTermShell/SilverLight/SilverLight/ISSH2ClientService.cs
using System;
using System.Net;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Documents;
using System.Windows.Ink;
using System.Windows.Input;
using System.Windows.Media;
using System.Windows.Media.Animation;
using System.Windows.Shapes;
using Weborb.Client;


namespace FlexTermShell.src
{
    public interface ISSH2ClientService
    {
        AsyncToken<SilverLight.dao.SSH2ResultDAO> tryConnectThenExec(SilverLight.dao.SSH2DAO ssh2dao);
    }    
}
